### PR TITLE
Fix lint errors in juju kubernetes master and e2e charms

### PR DESCRIPTION
**Pull request by ktsakalozos**
**What this PR does / why we need it**: Fixes style error in the Juju charms

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```
Code style fixes in Juju charms
```

### CODE REVIEW

**Comment by spiffxp** [comment]
@k8s-bot ok to test

**Comment by chuckbutler** [comment]
@k8s-bot bazel test this

@k8s-bot verify test this

File: cluster/juju/layers/kubernetes-master/Makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+.PHONY: clean
+clean:
+	@rm -rf .tox
+
+.PHONY: apt_prereqs
+apt_prereqs:
+	@# Need tox, but don't install the apt version unless we have to (don't want to conflict with pip)
+	@which tox >/dev/null || (sudo pip install tox)
+	@sudo apt-get install -y python-pip python-flake8 python3-flake8 flake8
```
Comment (chuckbutler) [inline cluster/juju/layers/kubernetes-master/Makefile]: is python pip required here? we're also calling pip install tox above with the possibility that pip isn't installed.

**Review by lazypower — CHANGES_REQUESTED** [review]
I had one check inline that I'd like clarification on before approval.

Otherwise this cleanup looks good. Thanks for the effort Kos!

**Comment by Cynerva** [comment]
Rather than including tox.ini in our charms, would it make more sense to open a bug against layer-snap? I don't think it's reasonable for layer-snap to break the charms that include it.

**Comment by thockin** [comment]
/approve

I have no context to weigh in on the questions

**Pull request merged**